lib/jsdom/living/helpers/geometry/dommatrix-init.js

```diff
@@ -0,0 +1,89 @@
+"use strict";
```

Comment:
I'd appreciate it if you could use more spec terminology in this file. For example, `getDOMMatrixDimension` really seems to be the last three steps in https://drafts.fxtf.org/geometry/#matrix-validate-and-fixup, while validateAliasedProps and fixupDOMMatrixInit together make up https://drafts.fxtf.org/geometry/#matrix-validate-and-fixup-2d. Let's try to organize the code as close to the spec as possible, and add links to the spec algorithm next to each function.

Reply:
sure thing!